### PR TITLE
dashboard modal issue fixed

### PR DESCRIPTION
**Pull request by tanjimhasan**

### CODE REVIEW

**Comment by kowsar89** [comment]
### Issue Details
dashboard button
pay now and promote button doens't work 
https://ibb.co/jk803Jm

File: assets/src/js/public/atmodal.js [hunk 1]
```diff
@@ -1,65 +1,67 @@
 /*
-    Name:  ATModal
-    Version: 1.0
-    Author: SovWare
-    Author URI: http://sovware.com
-*/
-
+		Name:  ATModal
+		Version: 1.0
+		Author: AazzTech
```
Comment (kowsar89) [inline assets/src/js/public/atmodal.js:4]: we don't use "AazzTech" anymore

**Review by kowsar89 — CHANGES_REQUESTED** [review]

**Comment by tanjimhasan** [comment]
Aztech removed

**Review by kowsar89 — APPROVED** [review]
works fine

**Pull request merged**